roles/pre-result/tasks/fiod.yml

```diff
@@ -0,0 +1,16 @@
+---
+
+- name: create directory
+  file:
+    path: "{{ results.path }}/fiod-{{ uuid }}"
+    state: directory
+
+- name: touch clat histogram summary files
```

Comment:
is this task just so you don't get an exception when you run fio-histo-log-pctiles.py?   If so you must might want to add a comment.  If your test is shorter than the log_hist_msec parameter then nothing gets logged, can this be flagged in the ansible script somehow as a warning?

Reply:
I added a check to the roles/result/tasks/fiod.yaml to only do processing/indexing if the run_time > sample rate